test/test_torch.py

```diff
@@ -7273,10 +7273,10 @@ def get_taylor_approximation(a, deg):
             return res
 
         def scale_square(a, deg):
-            if a.norm() < 1.0:
+            if a.abs().pow(2).sum().sqrt() < 1.0:
```

Comment:
nit: the abs here and below is not needed

Reply:
Needed for complex numbers.

Reply:
Ho right complex pow... Sounds good!